src/components/Events.module.scss

```diff
@@ -15,8 +17,22 @@
   width: 100%;
   gap: 30px;
   justify-content: space-between;
+  max-width: 100%;
 }
 
 .sport {
   font-weight: bold;
 }
+
+@media only screen and (max-width: $screen-md-min) {
+  .event {
+    margin-left: auto;
+    margin-right: auto;
+    height: 200px;
+    width: 343px;
+    border: 2px solid #333;
```

Comment:
Check how we approach colors in profiles. Add this to some color definition. Dont use random one-of colors, you will quickly loose track